nncf/common/tensor_statistics/collectors.py

```diff
@@ -349,6 +349,41 @@ def _max_aggregate(self):
         return self._tensor_processor.mean(stacked_max, axis=0)
 
 
+class MeanStatisticCollector(OfflineTensorStatisticCollector):
```

Comment:
I think we can come up with some kind of general solution here. From the first view: it's hard to understand which `*StatisticCollector`'s entity we should use for the extension with the new one; also there are too many different entities that overload themselves. This comment is general for `statistics.py` and `collectors.py`.

Reply:
I would suggest making it in a separate PR if there is any proposal about how to do this because the same changes should be applicable to all the collectors we have, right?

Reply:
Such change will affect the whole NNCF common part. Not sure that this PR is ready for that.